### ISSUE TITLE
Build fails in master due to coordinate repr changes

### ISSUE THREAD

**Issue by cdeil** [astropy/regions#75]
The `astropy/regions` tests on travis-ci in master are failing for the two builds using Astropy dev:

https://travis-ci.org/astropy/regions/jobs/175035799#L743
https://travis-ci.org/astropy/regions/jobs/175035801#L745

Those are doctest changed because apparently the repr changed for SkyCoord. Example:
```
_______________________________ test_circle_sky ________________________________
    def test_circle_sky():
        center = SkyCoord(3 * u.deg, 4 * u.deg)
        reg = CircleSkyRegion(center, 2 * u.arcsec)
    
>       assert str(reg) == 'CircleSkyRegion\ncenter: <SkyCoord (ICRS): (ra, dec) in deg\n    (3.0, 4.0)>\nradius: 2.0 arcsec'
E       assert 'CircleSkyReg...s: 2.0 arcsec' == 'CircleSkyRegi...s: 2.0 arcsec'
E         Skipping 54 identical leading characters in diff, use -v to show
E            deg
E         -     ( 3.,  4.)>
E         ?      -   -
E         +     (3.0, 4.0)>
E         ?        +    +
E           radius: 2.0 arcsec
```

This is annoying because it makes the tests fail for all open PRs.

@bsipocz @astrofrog or anyone: suggestion how to modify (or remove?) the tests to fix this?
(I *might* have time to fix this on the weekend or Monday if it's clear what to do, but not today.)

**Comment by astrofrog** [comment]
Fixed in #71

**Comment by cdeil** [comment]
Great!

If #71 goes in the next few days, fine, no need to split it out.

**Comment by astrofrog** [comment]
Fixed in #71
